### DISCUSSION TITLE
Which are the mandatory markers for a minimal USFM file?

### DISCUSSION THREAD

**Discussion by kavitharaju** [usfm-bible/tcdocs#29] [General]
As per the these tests, it expects `\id`, `\c`, `\p` and also `\v`. But text inside a paragraph is not mandatory
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/mandatory
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/paratextTests/NoErrorsEmptyBook/origin.usfm

But the following tests seem to contradict the above set rule
- Only `id` https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/paratextTests/NoErrorsShort/origin.usfm
- Empty `c` https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/special-cases/empty-c/origin.usfm
- Only `id` https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/special-cases/empty-book/origin.usfm

**Comment by KentSpiel** [comment]
We have a use case for creating a Paratext project with only Book introductions. We want to publish the book introductions on the DBL in order to establish copyright. In other words a project with no chapters. Would such a project be valid USFM?
